scripts/install-env.sh

```diff
@@ -40,6 +40,8 @@ if ! $CONTINUE; then
     exit
 fi
 
+set +e
+
 echo "${bold}install-env:${normal} Checking if Rust is installed..."
 command -v rustc >/dev/null 2>&1
 if [[ $? -eq 0 ]]; then
```

Comment:
Can we `set -e` again before running  ` curl https://sh.rustup.rs -sSf | sh`? If that fails, we should probably exit